docs/mgmt/polishing.md

```diff
@@ -225,6 +226,20 @@ rename-mapping:
 ```
 where the `EnumType` is the original name of the enumeration type in the **swagger**, and `enum_value` is the original name of the enumeration value in the **swagger**. In case we have spaces or other special character, you might need to use quotes to enclosing the key in this mapping to ensure everything is good without compile errors.
 
+## Rename a parameter in an operation
+
+There is a configuration that allows you to change the parameter names in an operation. For instance
+```yaml
+parameter-rename-mapping:
+  VirtualMachines_CreateOrUpdate:
+    name: vmName
+```
+This configuration is a two-level dictionary. The key of the first level dictionary is the operation id you would like to change. In the above sample, the generator will change the parameter names that match in the operation with operation id `VirtualMachines_CreateOrUpdate`.
+
+The value of the first level dictionary of this configuration is another dictionary, the original parameter name as its keys, and the desired value as its value. In the above sample, the generator will rename the parameter `name` to `vmName` in the operation `VirtualMachines_CreateOrUpdate`.
```

Comment:
`the original parameter name as its keys, and the desired value as its value` suggested to be `the original parameter names as its keys, and the desired parameter names as its values`.